### PR TITLE
Turn demo mode (html+CSS) in React app. import all assets and modify CSS

### PR DESCRIPTION
**Pull request by Olif51**

### CODE REVIEW

File: frontend/src/App.css [hunk 1]
```diff
@@ -27,8 +50,72 @@ h1, h2, h3, h4, h5, h6 {
...
+.propose {
+  margin: 0;
+  padding-left: 1rem;
+  font-size: 1.5rem;
+  text-shadow: 2px 0 0 red, 2px 2px 0 red, 0 2px 0 red, -2px 2px 0 red,
+  -2px 0 0 red, -2px -2px 0 red, 0 -2px 0 red, 2px -2px 0 red;
+}
+
+.firstChoice, .secondChoice, .yes, .no, .propose {
+  width: 45%;
```
Comment (rocambille) [inline frontend/src/App.css]: remove the width

File: frontend/src/App.jsx [hunk 2]
```diff
@@ -1,51 +1,33 @@
...
 import "./App.css";
-import Movie from "./components/Movie";
-import Filter from "./components/Filter";
-import Config from "./Config";
-
+import Choix1 from "./pages/Choix1";
+import Choix2 from "./pages/Choix2";
+/* import Indiana from "./pages/Indiana";
+import Rambo from "./pages/Rambo";
+import StarWars from "./pages/StarWars"; */
```
Comment (rocambille) [inline frontend/src/App.jsx]: dead code : remove it

File: frontend/src/pages/Choix2.jsx [hunk 3]
```diff
@@ -0,0 +1,37 @@
+function Choix2() {
+  return (
+    <section>
+      <h1>Avez-vous envie de vous évader ?</h1>
+      <br />
+      <br />
```
Comment (rocambille) [inline frontend/src/pages/Choix2.jsx]: remove the brs : that's a spacing problem

File: frontend/src/pages/Choix2.jsx [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+        </button>
+        <button
+          Type="button"
+          onClick="location.href='pres_star.html'"
+          className="secondChoice"
+        >
+          Blaster ?
+        </button>
+      </div>
+      <br />
```
Comment (rocambille) [inline frontend/src/pages/Choix2.jsx]: remove the br

File: frontend/src/pages/Indiana.jsx [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+
+        <button
+          type="button"
+          className="seen"
+          onClick="location.href='pres_rambo.html'"
+        >
+          Déja Vu
+        </button>
+        <br />
+        <br />
```
Comment (rocambille) [inline frontend/src/pages/Indiana.jsx]: remove the brs

File: frontend/src/pages/Rambo.jsx [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+
+        <button
+          type="button"
+          className="seen"
+          onClick="location.href='pres_indiana.html'"
+        >
+          Déja Vu
+        </button>
+        <br />
+        <br />
```
Comment (rocambille) [inline frontend/src/pages/Rambo.jsx]: remove the brs

File: frontend/src/pages/StarWars.jsx [hunk 7]
```diff
@@ -0,0 +1,36 @@
...
+
+        <button
+          type="button"
+          className="seen"
+          onClick="location.href='pres_indiana.html'"
+        >
+          Déja Vu
+        </button>
+        <br />
+        <br />
```
Comment (rocambille) [inline frontend/src/pages/StarWars.jsx]: remove the brs

**Review by rocambille — CHANGES_REQUESTED** [review]

File: frontend/src/App.css (hunk 1 above)
Comment (Olif51) [reply frontend/src/App.css]: fixed :)

File: frontend/src/App.jsx (hunk 2 above)
Comment (Olif51) [reply frontend/src/App.jsx]: fixed :)

File: frontend/src/pages/Choix2.jsx (hunk 4 above)
Comment (Olif51) [reply frontend/src/pages/Choix2.jsx]: fixed :p

File: frontend/src/pages/Indiana.jsx (hunk 5 above)
Comment (Olif51) [reply frontend/src/pages/Indiana.jsx]: br removed ^^'

File: frontend/src/pages/Rambo.jsx (hunk 6 above)
Comment (Olif51) [reply frontend/src/pages/Rambo.jsx]: done

File: frontend/src/pages/StarWars.jsx (hunk 7 above)
Comment (Olif51) [reply frontend/src/pages/StarWars.jsx]: done

**Pull request merged**